helpers/EnumHelper.cs

```diff
@@ -4,24 +4,19 @@ namespace BerryLoaderNS
 {
 	public static class EnumHelper
 	{
+		public static T ToTCardType<T>(string str)
```

Comment:
This one is for some DRY principle : "Don't Repeat Yourself".
This may be for 3 nowadays, but next version it'll be 4 or 99, this generic `ToTCardType<T>` would handle it nicely.
Maybe making it `private` would suit you better?

Reply:
I'm actually not great at namings, is `EnumParseStrToT` a valid one?
I already used `StringToT` at work (API deserialization), that feeled weird to me here.
Which one would you prefer?

Reply:
*decision pending tag*